### ISSUE TITLE
Variables unavailable in trigger conditions

### ISSUE THREAD

**Issue by rfkat** [sqshq/sampler#107]
System: Ubuntu 20.04.3 LTS

It seems impossible to refer to variables in trigger conditions. That would be helpful wrt defining global trigger values *once*, at the top of the YAML file.

For now, I've resorted to using named pipes as a workaround, but that's quite clumsy.

**Comment by sqshq** [comment]
I agree it would be useful, might implement that at some point. Feel free to contribute if interested.

**Comment by rfkat** [comment]
I might, once I have mastered go (my active developer days are over, since a decade ;)).
